### DISCUSSION TITLE
index sibling module considered parallel route, not nested

### DISCUSSION THREAD

**Discussion by akomm** [remix-run/remix#1448] [Q&A]
Example routes:

- items/index.tsx
- items/create.tsx

I need to create `items/index.tsx` as `items.tsx` for the `create.tsx` to be considered as child route.

I know you can place route modules as siblings for not sharing layout, but I wonder if this behavior for `index` is intentional.

**Comment by sergiodxa** [comment]
If want to have a route nested inside another you need this setup
```
routes/parent/child.tsx
routes/parent.tsx
```
The parent is a created as usual and the nested is inside a folder of the same name of the parent.

Creating and index.tsx inside the folder will match at the /parent url, and the rest will match at /parent/child.

**Reply by akomm** [reply]
@sergiodxa Thanks. I just wrote myself what you explain to me, I asked whether this behavior is also intentional for `index`.
